pennylane/fermi/fermionic.py

```diff
@@ -324,6 +324,111 @@ def to_mat(self, n_orbitals=None, format="dense", buffer_size=None):
             wire_order=list(range(largest_order)), format=format, buffer_size=buffer_size
         )
 
+    def commute(self, source, target):
```

Comment:
I think `commute` might be a misleading name for this method. Might want to reconsider using something like `permute_terms` or `swap_operators` with a signature like `func(self, index1, index2)`.

Reply:
`shift_operator` sounds good to me! 👍

Reply:
Yes, "shift" is the correct keyword.